[2025-06-23T05:09:53.730+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-23T05:09:53.808+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_training.cleanup_resources manual__2025-06-22T22:46:21.358365+00:00 [queued]>
[2025-06-23T05:09:53.837+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_training.cleanup_resources manual__2025-06-22T22:46:21.358365+00:00 [queued]>
[2025-06-23T05:09:53.840+0000] {taskinstance.py:2867} INFO - Starting attempt 2 of 2
[2025-06-23T05:09:53.868+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): cleanup_resources> on 2025-06-22 22:46:21.358365+00:00
[2025-06-23T05:09:53.891+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection_training', 'cleanup_resources', 'manual__2025-06-22T22:46:21.358365+00:00', '--job-id', '186', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_training_dag.py', '--cfg-path', '/tmp/tmpqq91aj93']
[2025-06-23T05:09:53.895+0000] {standard_task_runner.py:105} INFO - Job 186: Subtask cleanup_resources
[2025-06-23T05:09:53.889+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3658) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-23T05:09:53.897+0000] {standard_task_runner.py:72} INFO - Started process 3666 to run task
[2025-06-23T05:09:53.999+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection_training.cleanup_resources manual__2025-06-22T22:46:21.358365+00:00 [running]> on host ec3b7e32037e
[2025-06-23T05:09:54.373+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Jasjot Parmar' AIRFLOW_CTX_DAG_ID='fraud_detection_training' AIRFLOW_CTX_TASK_ID='cleanup_resources' AIRFLOW_CTX_EXECUTION_DATE='2025-06-22T22:46:21.358365+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-22T22:46:21.358365+00:00'
[2025-06-23T05:09:54.374+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-06-23T05:09:54.405+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-06-23T05:09:54.409+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'rm -f /app/tmp/*.pkl']
[2025-06-23T05:09:54.580+0000] {subprocess.py:99} INFO - Output:
[2025-06-23T05:09:54.604+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-06-23T05:09:54.770+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-06-23T05:09:54.772+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fraud_detection_training, task_id=cleanup_resources, run_id=manual__2025-06-22T22:46:21.358365+00:00, execution_date=20250622T224621, start_date=20250623T050953, end_date=20250623T050954
[2025-06-23T05:09:54.901+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-23T05:09:54.925+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-23T05:09:54.932+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
